### PR TITLE
stagehand.act -> page.act

### PR DESCRIPTION
**Pull request by kamath**
`stagehand.act` forwards the act to `stagehand.page` for now with a deprecation warning. 

waiting for evals to pass here before a fast-follow PR to change evals to reference `stagehand.page.act` as well

### CODE REVIEW

**Comment by kamath** [comment]
commenting here to link a check of [all the important evals passing](https://github.com/browserbase/stagehand/actions/runs/12451166770). Any changes below this are non-code changes

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -332,10 +335,7 @@ jobs:
           if [ -f eval-summary.json ]; then
             combination_score=$(jq '.categories.combination' eval-summary.json)
             echo "Combination category score: $combination_score%"
-            if (( $(echo "$combination_score < 85" | bc -l) )); then
```
Comment (kamath) [inline .github/workflows/ci.yml:336]: let's stop failing these and let's find a better spot to call passing

Comment (seanmcguire12) [reply .github/workflows/ci.yml:336]: what does "find a better spot to call passing" mean?

Comment (seanmcguire12) [reply .github/workflows/ci.yml:336]: id rather take 4o-mini out of CI eval models than completely remove combos from CI

**Review by seanmcguire12 — APPROVED** [review]

**Pull request merged**